js/ui/grid_core/ui.grid_core.editing.js

```diff
@@ -2911,7 +2911,7 @@ module.exports = {
                         editingController.closeEditCell();
                     }
 
-                    return eventName === startEditAction && allowEditing && editingController.editCell(e.rowIndex, columnIndex) || editingController.isEditRow(e.rowIndex);
+                    return eventName === startEditAction && allowEditing && editingController.editCell(e.rowIndex, columnIndex) || allowEditing && editingController.isEditRow(e.rowIndex);
```

Comment:
Suppose this conditions needs to be refactored

Reply:
Simplified the condition